File: app/src/main/java/com/yawar/memo/views/VideoActivity.java
package com.yawar.memo.views;

import androidx.appcompat.app.AppCompatActivity;

import android.media.MediaPlayer;
import android.net.Uri;
import android.os.Bundle;
import android.view.WindowManager;
import android.widget.MediaController;
import android.widget.Toast;
import android.widget.VideoView;

import com.yawar.memo.R;

public class VideoActivity extends AppCompatActivity {
    MediaController mediaControls;
    VideoView videoView;
    Bundle bundle;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,
                WindowManager.LayoutParams.FLAG_FULLSCREEN);
        setContentView(R.layout.activity_video);
        Bundle bundle = getIntent().getExtras();
        String path = bundle.getString("path","1");
        videoView = findViewById(R.id.simpleVideoView);
        videoView.requestFocus();
        videoView.setOnPreparedListener(new MediaPlayer.OnPreparedListener() {
            @Override
            public void onPrepared(MediaPlayer mediaPlayer) {
                mediaPlayer.setOnVideoSizeChangedListener(new MediaPlayer.OnVideoSizeChangedListener() {
                    @Override
                    public void onVideoSizeChanged(MediaPlayer mp, int width, int height) {
                        /*
                         * add media controller
                         */
                        mediaControls = new MediaController(VideoActivity.this);
                        videoView.setMediaController(mediaControls);
                        /*
                         * and set its position on screen
                         */
                        mediaControls.setAnchorView(videoView);
                    }
                });
            }
        });
//        mediaControls = new MediaController(this);
        videoView.setMediaController(mediaControls);
//        mediaControls.setAnchorView(videoView);
        videoView.setVideoURI(Uri.parse(path));
//                // start a video
        videoView.start();
        videoView.setOnCompletionListener(new MediaPlayer.OnCompletionListener() {
            @Override
            public void onCompletion(MediaPlayer mp) {
                finish();
//                Toast.makeText(getApplicationContext(), "Thank You...!!!", Toast.LENGTH_LONG).show(); // display a toast when an video is completed
            }
        });
        videoView.setOnErrorListener(new MediaPlayer.OnErrorListener() {
            @Override
            public boolean onError(MediaPlayer mp, int what, int extra) {
                finish();

//                Toast.makeText(getApplicationContext(), "Oops An Error Occur While Playing Video...!!!", Toast.LENGTH_LONG).show(); // display a toast when an error is occured while playing an video
                return false;
            }
        });
    }
}

File: app/src/main/java/com/yawar/memo/views/VideoActivity.kt
package com.yawar.memo.views

import android.media.MediaPlayer.*
import android.net.Uri
import android.os.Bundle
import android.view.WindowManager
import android.widget.MediaController
import android.widget.VideoView
import androidx.appcompat.app.AppCompatActivity
import com.yawar.memo.R

class VideoActivity : AppCompatActivity() {
    var mediaControls: MediaController? = null
    lateinit var videoView: VideoView
    var bundle: Bundle? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        window.setFlags(
            WindowManager.LayoutParams.FLAG_FULLSCREEN,
            WindowManager.LayoutParams.FLAG_FULLSCREEN
        )
        setContentView(R.layout.activity_video)
        val bundle = intent.extras
        val path = bundle!!.getString("path", "1")
        videoView = findViewById(R.id.simpleVideoView)
        videoView.requestFocus()
        videoView.setOnPreparedListener(OnPreparedListener { mediaPlayer ->
            mediaPlayer.setOnVideoSizeChangedListener { mp, width, height ->
                mediaControls = MediaController(this)
                videoView.setMediaController(mediaControls)
                mediaControls!!.setAnchorView(videoView)
            }
        })
        videoView.setMediaController(mediaControls)
        videoView.setVideoURI(Uri.parse(path))
        videoView.start()
        videoView.setOnCompletionListener(OnCompletionListener { finish() })
        videoView.setOnErrorListener(OnErrorListener { mp, what, extra ->
            finish()
            false
        })
    }
}